Resources/Prototypes/_RMC14/Catalog/Fills/Crates/gear.yml

```
@@ -1,4 +1,26 @@
 ﻿- type: entity
+  parent: RMCCrateAmmo
+  id: RMCCrateGearBinoculars
+  name: mixed binoculars crate (2x per, 6x total)
+  components:
+  - type: StorageFill
+    contents:
+    - id: RMCBinoculars
+      amount: 2
+    - id: RMCLaserDesignator # TODO add 2 rangefinders and reduce the amount of this to 2
+      amount: 4
+
+- type: entity
+  parent: RMCCrateAmmo
+  id: RMCCrateGearPackFlare
+  name: flare packs crate (x20)
+  components:
+  - type: StorageFill
+    contents:
+    - id: CMPackFlare
+      amount: 20
```

This is supposed to be 2 boxes of flare packs
```suggestion
    - id: RMCBoxPackFlareBase
      amount: 2
```